lib/Graph/VerifierHelper.cpp

```diff
@@ -69,8 +69,12 @@ bool glow::checkType(NodeValue A, ElemKind expectedType, const Node *parent) {
 
 bool glow::checkSameIsQuantized(const TypeRef A, const TypeRef B,
                                 const Node *parent) {
-  return expectCompareTrue("Mismatching isQuantized", A->isQuantizedType(),
-                           B->isQuantizedType(), parent);
+  bool ret = expectCompareTrue("Mismatching isQuantized", A->isQuantizedType(),
+                               B->isQuantizedType(), parent);
+  if (!ret) {
+    printf("now quantization has issue\n");
```

Comment:
I see `llvm::errs()` is used more frequently in the code, maybe prefer that?
Also this error message doesn't provide much information

Reply:
Thanks! Actually, these lines are used for testing and should be removed :)